lib/media/ttml_text_parser.js

```diff
@@ -160,12 +160,13 @@ shaka.media.TtmlTextParser.percentValues_ = /^(\d{1,2}|100)% (\d{1,2}|100)%$/;
  */
 shaka.media.TtmlTextParser.lineAlignConvTable_ = {
```

Comment:
let's call it "textAlignToLineAlign_" instead so it's clear what we are mapping.